custom_components/localtuya/cover.py

```diff
@@ -26,14 +34,23 @@
 DEFAULT_OPEN_CMD = "on"
 DEFAULT_CLOSE_CMD = "off"
 DEFAULT_STOP_CMD = "stop"
```

Comment:
You did remove the stop command, so you should remove this too in that case. I think `stop_cmd` is in the example YAML in `__init__.py` as well, which you can remove.